packages/x-components/src/plugins/devtools/wiring.devtools.ts

```diff
@@ -0,0 +1,127 @@
+import { Dictionary, forEach, reduce } from '@empathyco/x-utils';
+import { CustomInspectorNode, DevtoolsPluginApi, InspectorNodeTag } from '@vue/devtools-api';
+import { XEvent } from '../../wiring/events.types';
+import { filter } from '../../wiring/wires.operators';
+import { AnyWire, Wiring } from '../../wiring/wiring.types';
+import { XModuleName } from '../../x-modules/x-modules.types';
+import { hslToHex, moduleColors } from './utils';
+
+/** The full list of wiring nodes for the inspector. */
+const WiringNodes: Partial<Record<XEvent, CustomInspectorNode[]>> = {};
```

Comment:
Should these consts be in uppercase to follow the naming convention?

Reply:
Again, why are we using upper camel case? 😅